integration_tests/mockApis/laoStatus.ts

```diff
@@ -0,0 +1,18 @@
+import { SuperAgentRequest } from 'superagent'
+import { stubForLaoStatus } from './wiremock'
+
+export default {
+  stubForLaoStatus: ({ crn, response }): SuperAgentRequest => {
+    return stubForLaoStatus({
+      request: {
+        method: 'GET',
+        urlPattern: `/cases/unallocated/${crn}/cases/restricted`,
```

Comment:
This endpoint should be **/cases/unallocated/{crn}/restricted**

Reply:
fixed